lib/services/network/repository.dart

```diff
@@ -16,7 +16,9 @@ import 'api_provider.dart';
 
 abstract class RepositoryAPI {
   Future<User> getIdentity();
-  Future<LoginResponsePodo> login(LoginRequestPodo loginRequestPodo);
+
+  Future<LoginResponseDto> login(LoginRequest loginRequestPodo);
```

Comment:
Why are there extra spaces?

Reply:
Because of the android studio auto reformating

Reply:
If this is the only function with spaces, remove them OR insert spaces between each function.